src/cosmograph/__init__.py

```
@@ -24,7 +24,7 @@
 """
 
 from cosmograph.base import cosmo, base_cosmo
```

Ruff (0.8.2)

`cosmograph.base.cosmo` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)